src/Discord.Net.Commands/Builders/ModuleClassBuilder.cs

```diff
@@ -193,23 +199,47 @@ private static void BuildParameter(ParameterBuilder builder, System.Reflection.P
                 }
             }
 
-            var reader = service.GetTypeReader(paramType);
-            if (reader == null)
+            if (builder.TypeReader == null)
             {
-                var paramTypeInfo = paramType.GetTypeInfo();
-                if (paramTypeInfo.IsEnum)
+                var readers = service.GetTypeReaders(paramType);
+                var reader = readers?.FirstOrDefault();
```

Comment:
Is it safe to always assume the first reader is the default? Maybe we should have a separate field for the default/global reader?